src/components/ui/Panel.jsx

```diff
@@ -330,7 +330,9 @@ class Panel extends React.Component {
             {floatingItemsRight && size !== 'maximized' && (
               <FloatingItems position="right" items={floatingItemsRight} />
             )}
-            {onClose && <CloseButton onClick={onClose} className="panel-close" />}
+            {onClose && !isMobileDevice() && (
```

Comment:
No need for the imperative `isMobileDevice` function, we are already in a `DeviceContext` here, `isMobile` is already available.